arbitrum-docs/stylus/reference/opcode-hostio-pricing.md

```diff
@@ -1,167 +1,171 @@
 ---
-title: 'Opcode and Host I/O Pricing'
-sidebar_label: 'Opcode and Host I/O Pricing'
-description: 'A reference of how much Opcodes and Host I/Os cost in Stylus, with measurements in ink and gas.'
+title: 'Opcode and host I/O pricing'
```

Comment:
sentence-casing as a general pattern, per [style guide](https://docs.arbitrum.io/for-devs/contribute#style-conventions)